tensorflow/tools/dist_test/README.md

```diff
@@ -122,3 +122,32 @@ servers. For example:
 
 See [Kubernetes kubectl documentation](http://kubernetes.io/docs/user-guide/kubectl-overview/)
 for more details.
+
+**Create allreduce based Tensorflow k8s deployment**
```

Comment:
Sorry for nitpicking. Could you please change all occurrences of `allreduce based` to `allreduce-based`?

Reply:
I changed all allreduce based to allreduce-based.